src/lib/search-and-replace.js

```diff
@@ -123,6 +134,8 @@ export const searchAndReplace = async (
 	{ isImport = true, inPlace = false, output = process.stdout }: SearchReplaceOptions,
 	binary: string | null = null
 ): Promise<SearchReplaceOutput> => {
+	console.log( `${ 'Starting Search and Replace...' }` );
```

Comment:
```suggestion
	console.log( 'Starting Search and Replace...' );
```

Reply:
Oop, I initially added a `chalk.cyan()` there but took it out, and forgot to change this back! Fixed!